Makefile

```diff
@@ -24,7 +24,7 @@ ifeq ($(DEV),true)
 endif
 
 serve:
-	docker run --rm -ti -e JEKYLL_UID=`id -u` -p 4000:4000 -v $$PWD:/srv/jekyll jekyll/jekyll:$(JEKYLL_VERSION) jekyll serve --incremental $(CONFIG)
+	docker run --rm -ti -e JEKYLL_UID=`id -u` -p 4000:4000 -v $$PWD:/srv/jekyll jekyll/jekyll:$(JEKYLL_VERSION) jekyll serve --incremental --no-watch $(CONFIG)
```

Comment:
From my view, I like this flag but I know some people have had trouble with it.

Reply:
I can remove after, just needed it to view my changes :)